src/utils/text.js

```diff
@@ -39,4 +39,24 @@ function formatKeyShortcut(shortcut: string): string {
     .replace(/Shift\+/g, "Shift ");
 }
 
-export { formatKeyShortcut };
+/**
+ * Truncates the received text to the maxLength in the format:
+ * Original: 'this is a very long text and ends here'
+ * Truncated: 'this is a ver...nd ends here'
+ * @param {String} text - Original text
+ * @param {Number} maxLength - Max allowed length
+ * @memberof utils/text
+ * @static
+ */
+function truncateText(text: string, maxLength: number): string {
```

Comment:
i kinda think of truncating at the end as the default. Do want to name this function `truncateMiddleText`

Also do we need the word text, that makes me think of source text

Reply:
I have updated the name as suggested. :)